bin/rspec.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -
+
+echo "Running R-Spec..."
+
+RAILS_ENV=test bundle exec rspec spec
```

Comment:
is there a benefit to having this wrapper script vs just running `bundle exec rspec`? in console?

Reply:
Because I'm lazy and I just want to run `./bin/spec.sh` when I'm on a bash console inside docker under `/app/src`. Just saves typing.